_includes/catalogpage.html

```diff
@@ -15,7 +15,7 @@
         {% endfor %}
     </ul>
 
-    <p>See more widgets in the <a href="..">Flutter widget catalog</a>.</p>
+    <p>See more widgets in the <a href="/widgets/widgetindex/">Flutter widget catalog</a>.</p>
```

Comment:
https://flutter.io/widgets/, the grid of categories. (As opposed to widgetindex, the grid of all widgets.)